build/commands/lib/build_fuzzer.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+const build = require('./build')
+
+const buildFuzzer = (fuzzer_test_target, options) => {
+  options.use_libfuzzer = true
+  options.is_asan = true
+  options.C = 'Fuzzer'
```

Comment:
this is going to be problematic for non-host arch/os because they will all try to build in the same output dir

Reply:
or does this only run/build as host_os? If so, does it use that explicitly so passing target_os/target_arch won't break it?

Reply:
Removed C option.